codecov.yml

```
@@ -0,0 +1,23 @@
+comment:
+  layout: newheader, reach, files, components, diff, flags # show component info in the PR comment
+  hide_comment_details: true  # hide the comment details (e.g. coverage targets) in the PR comment
+
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 30%
+        removed_code_behavior: adjust_base
+    patch:
+      default:
+        target: 40%
+        threshold: 40%
```

How about being a bit more strict? (it won't prevent merges but it's "forcing" us to always test everything we add). 

```suggestion
        target: 80%
        threshold: 1%
        removed_code_behavior: adjust_base
    patch:
      default:
        target: 90%
        threshold: 10%
```

---

For now I would like to keep the current intentional values as we need to do a lot for these values. But agreed we need better values.

---

(this is a forked repo, needs some work before we can use those strict values)

---

I took your patch suggestion :)